### DISCUSSION TITLE
cant browse playlist tracks

### DISCUSSION THREAD

**Discussion by Iqbal198** [punxaphil/custom-sonos-card#480] [Q&A]
i have playlists added into the card by favouriting sonos playlists within the windows app.

i can play and shuffle the playlists in the card but cant find any way to look inside the playlist and select tracks within.

any help would be greatly appreciated.

**Comment by punxaphil** [comment]
I would like to have this too, but the Sonos integration doesn't support it, so not much I can do 😢 
https://www.home-assistant.io/integrations/sonos/

**Comment by punxaphil** [comment]
@Iqbal198 
There is now queue/playlist added in the beta: https://github.com/punxaphil/custom-sonos-card/issues/355#issuecomment-2440158692
